cinema/tests/test_order_api.py

```diff
@@ -73,12 +71,8 @@ def test_movie_session_detail_tickets(self):
             f"/api/cinema/movie_sessions/{self.movie_session.id}/"
         )
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertEqual(
-            response.data["taken_places"][0]["row"], self.ticket.row
-        )
-        self.assertEqual(
-            response.data["taken_places"][0]["seat"], self.ticket.seat
-        )
+        self.assertEqual(response.data["taken_places"][0]["row"], self.ticket.row)
+        self.assertEqual(response.data["taken_places"][0]["seat"], self.ticket.seat)
 
     def test_movie_session_list_tickets_available(self):
         response = self.client.get(f"/api/cinema/movie_sessions/")
```

Comment:
In `test_movie_session_list_tickets_available`, ensure that the logic for calculating `tickets_available` is correctly implemented in the API. This test assumes that one ticket has been sold, hence the subtraction of 1 from the cinema hall capacity.